docs/docusaurus/docs/snippets/result_format.py

```diff
@@ -1,16 +1,6 @@
-from typing import Any, Dict, List
-
 import pandas as pd
 
 import great_expectations as gx
-from great_expectations.checkpoint import Checkpoint
```

Comment:
REWRITE - we may want to write a new version of this test. ID/PK integration with V1 Checkpoints should be discussed and appropriately tested